src/integrations/room-presence/room-presence.proxy.ts

```diff
@@ -15,6 +19,9 @@ export class RoomPresenceDeviceTrackerProxyHandler
     if (p === 'state') {
       this.deviceTracker.state = value != STATE_NOT_HOME;
     }
+    if (p === 'batteryLevel') {
```

Comment:
Could be an `else if` or `switch` for more clarity, but is also fine as is if you want to keep it that way

Reply:
Done.